File: TPoint/TPoint/Domain/Entities/Player.swift
//
//  Player.swift
//  TPoint
//

import Foundation

struct Player: Hashable, Codable {
    let team: TeamType
    let position: Int  // 0 or 1 (팀 내 순서)

    var displayName: String {
        "\(team.shortName)\(position + 1)"
    }

    static let allPlayers: [Player] = [
        Player(team: .teamA, position: 0),
        Player(team: .teamA, position: 1),
        Player(team: .teamB, position: 0),
        Player(team: .teamB, position: 1)
    ]
}

File: TichuTallyAndroid/app/src/main/java/com/tichutally/app/domain/model/Player.kt
package com.tichutally.app.domain.model

import kotlinx.serialization.Serializable

@Serializable
data class Player(
    val team: TeamType,
    val position: Int  // 0 or 1
) {
    val displayName: String
        get() = "${team.shortName}${position + 1}"

    companion object {
        val allPlayers = listOf(
            Player(TeamType.TEAM_A, 0),
            Player(TeamType.TEAM_A, 1),
            Player(TeamType.TEAM_B, 0),
            Player(TeamType.TEAM_B, 1)
        )
    }
}
